testing/build.gradle

```diff
@@ -82,6 +83,8 @@ dependencies {
       project(":domain"),
       project(":utility"),
   )
+  implementation 'androidx.appcompat:appcompat:1.1.0'
+  implementation "org.jetbrains.kotlin:kotlin-stdlib:$kotlin_version"
```

Comment:
Please add these alphabetically to the implementation block above.

Reply:
Done.